test/test_parse_psse.jl

```diff
@@ -23,10 +23,11 @@ end
 
 @testset "PSSE Component Parsing" begin
     @info "Testing Load Parsing"
-    sys = build_system(PSYTestSystems, "psse_240_parsing_sys") # current/imedance_power read in natural units during parsing
+    sys = build_system(PSYTestSystems, "psse_240_parsing_sys") # current/impedance_power read in natural units during parsing
     @test get_current_active_power(get_component(StandardLoad, sys, "load10021")) == 223.71
     @test get_impedance_reactive_power(get_component(StandardLoad, sys, "load10021")) ==
           583.546
+    PSB.clear_serialized_systems("psse_Benchmark_4ger_33_2015_sys")
```

Comment:
Given https://github.com/NREL-Sienna/PowerSystemCaseBuilder.jl/pull/115, is this necessary anymore?

Reply:
I can try to remove it. We added it because it was failing for some reason.